packages/server-wallet/src/__test-with-peers__/crash-tolerance/crash-tolerance.test.ts

```diff
@@ -1,127 +0,0 @@
-import {
```

Comment:
Just to clarify, this test case is now covered by the cases
```
  {dropRate: 0.1, meanDelay: 50, closer: 'A'},
  {dropRate: 0.1, meanDelay: 50, closer: 'B'},
```
in the direct funding test?

Reply:
I think the `crash-tolerance`  test is slightly different since it is focusing on "restarting" the engine and make sure it can keep on executing. The test cases in this PR are just testing messages being dropped/delayed.